test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -69,6 +69,18 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(doc.Id, id);
         }
 
+        protected async Task NotificationHubTest()
+        {
```

Comment:
We could have the message sent to you Brett, then you could manually increment a value in table store, and she could poll for that in the test. You'd have to remain responsive 24/7 though so the tests run reliably.